protected/models/Patient.php

```diff
@@ -166,6 +166,12 @@ public function relations()
             'allergies' => array(self::MANY_MANY, 'Allergy', 'patient_allergy_assignment(patient_id, allergy_id)',
                 'alias' => 'patient_allergies',
                 'order' => 'patient_allergies.name', ),
+            'medications' => array(self::MANY_MANY, '\OEModule\OphCiExamination\models\HistoryMedicationsEntry', 'patient_medication_assignment(patient_id, id)',
+                'alias' => 'patient_medications',
+                'order' => 'patient_medications.medication_name', ),
+            'diagnoses' => array(self::MANY_MANY, '\OEModule\OphCiExamination\models\OphCiExamination_Diagnosis', 'patient_diagnosis_assignment(patient_id, id)',
+                'alias' => 'patient_diagnoses',
+                'order' => 'patient_diagnoses.principal DESC', ),
```

Comment:
You might also want to order by something other than whether they are principal diagnoses

Reply:
Not sure how that would be structured in this section though.